packages/vitest-pool-workers/src/worker/lib/cloudflare/test-runner.ts

```diff
@@ -209,15 +209,6 @@ export default class WorkersTestRunner extends VitestTestRunner {
 		initialState.current = this.state.current;
 	}
 
-	async onBeforeRunFiles() {
-		if (DEBUG) {
-			__console.log("onBeforeRunFiles");
-			await scheduler.wait(100);
-		}
-
-		resetMockAgent(fetchMock);
```

Comment:
We will need to figure out where to reset the MockAgent if `onBeforeRunFiles` is removed